flink-ml-lib/src/main/java/org/apache/flink/ml/feature/maxabsscaler/MaxAbsScaler.java

```diff
@@ -0,0 +1,201 @@
+                output.collect(new StreamRecord<>(maxVector));
+            }
+        }
+
+        @Override
+        public void processElement(StreamRecord<Vector> streamRecord) {
+            Vector currentValue = streamRecord.getValue();
+            if (currentValue == null) {
+                throw new RuntimeException("Input column data cannot be null.");
+            }
+            if (maxVector == null) {
+                int vecSize = currentValue.size();
+                maxVector = new DenseVector(vecSize);
+                if (currentValue instanceof DenseVector) {
+                    System.arraycopy(
+                            ((DenseVector) currentValue).values, 0, maxVector.values, 0, vecSize);
+                } else {
+                    int[] indices = ((SparseVector) currentValue).indices;
+                    double[] values = ((SparseVector) currentValue).values;
+                    for (int i = 0; i < indices.length; ++i) {
+                        maxVector.values[indices[i]] =
+                                Math.max(maxVector.values[indices[i]], values[i]);
+                    }
+                }
+            } else {
+                Preconditions.checkArgument(
+                        currentValue.size() == maxVector.size(),
+                        "CurrentValue should has same size with maxVector.");
+                if (currentValue instanceof DenseVector) {
+                    double[] values = ((DenseVector) currentValue).values;
+                    for (int i = 0; i < currentValue.size(); ++i) {
+                        maxVector.values[i] = Math.max(maxVector.values[i], values[i]);
+                    }
+                } else if (currentValue instanceof SparseVector) {
+                    int[] indices = ((SparseVector) currentValue).indices;
+                    double[] values = ((SparseVector) currentValue).values;
+                    for (int i = 0; i < indices.length; ++i) {
+                        maxVector.values[indices[i]] =
+                                Math.max(maxVector.values[indices[i]], values[i]);
+                    }
+                } else {
+                    throw new RuntimeException(
+                            "Input column type must be SparseVector or DenseVector. ");
+                }
+            }
+        }
+
+        @Override
+        @SuppressWarnings("unchecked")
+        public void initializeState(StateInitializationContext context) throws Exception {
+            super.initializeState(context);
+            maxState =
+                    context.getOperatorStateStore()
+                            .getListState(
+                                    new ListStateDescriptor<>(
+                                            "maxState", TypeInformation.of(DenseVector.class)));
+            OperatorStateUtils.getUniqueElement(maxState, "maxState").ifPresent(x -> maxVector = x);
+        }
+
+        @Override
+        @SuppressWarnings("unchecked")
```

Comment:
Some SuppressWarnings like this are redundant. Could you please check and remove them?